src/app/actions/searchResult.ts

```diff
@@ -5,7 +5,12 @@ import { PaginationParams } from '@/lib/pagination';
 import SearchResultQuery from '@/queries/SearchResultQuery';
 import { findById } from '@/services/searchResultEntity';
 
-export async function listSearchResults(paginationParams: PaginationParams, keyword?: string) {
+// To Do create a interface for arguments greater than 3
```

Comment:
Wrong expected casing:

```suggestion
// TOTO: create an interface for arguments greater than 3
```

At the same time, this comment is not that useful 😅